pkgs/by-name/ta/tabby-terminal/package.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchurl, pkgs, lib, alsa-lib }:
+
+stdenv.mkDerivation rec {
+  name = "tabby-terminal";
+  version = "1.0.207";
+  src = fetchurl {
+    url = "https://github.com/Eugeny/tabby/releases/download/v${version}/tabby-${version}-linux-x64.deb";
+    hash = "sha256-PTTkL3+mYb7KM8fDUmgCuAF2lU88fYOstGWp/O5WZas=";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    dpkg
+    makeWrapper
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    dpkg -x $src .
+
+    runHook postUnpack
+  '';
+
+  buildInputs = with pkgs; [
+    glib
+    nss
+    nspr
+    atk
+    cups
+    dbus
+    libdrm
+    gtk3
+    pango
+    cairo
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    libxkbcommon
+    mesa
+    expat
+    xorg.libxcb
+    alsa-lib
+    libGL
+    libsecret
+    musl
```

Comment:
That executable is dynamically linking `musl`? Are you sure?

Reply:
Yes. Without it, the app launches but the loading bar never finishes.